### ISSUE TITLE
【BUG feedback】The 5.0.97 beta will disable the ZoteroQuickLook plugin

### ISSUE THREAD

**Issue by West-brick** [mronkko/ZoteroQuickLook#41]
QuickLook plugin is very easy to use on the Mac，but the 5.0.97 beta will disable the QuickLook plugin，the 5.096 beta version can still be used. I suspect that starting from the 5.097 beta version, the way to open the file is different. Previously, it was opened with a path, but now it is opened with a url.
I hope the authors have time to solve this problem, because this is a very easy to use and convenient function, thank you

**Comment by wanghongtaocn** [comment]
I have the same problem. Quicklook is a very nice fucntion. Hope it could be fixed.

**Comment by bwiernik** [comment]
This is the 5.0.97 beta of Zotero?

**Comment by West-brick** [comment]
Yes, I'm MacOS Big Sur 11.3. I used to run Zotero beta, but once Zotero automatically updated to Zotero 5.0.97-beta.33+ FDCD4E51C, QuickLook didn't work. Later I tried to downsize the version to Zotero 5.0.96.2, and it worked, but because it was not the latest beta, I lost the built-in PDF reader and global search, etc., and my friends also had a lot of such problems, so I was not sure whether the Windows system would work properly. However, QuickLook is not working on the Mac side because of the recent beta update. I hope it can be fixed. This preview function is really efficient，thank you

**Comment by wanghongtaocn** [comment]
cannot work on win10, either.

**Comment by enozkan** [comment]
https://forums.zotero.org/discussion/90505/bug-feedback-the-5-0-97-beta-will-disable-the-zoteroquicklook-plugin

**Comment by satan-j** [comment]
same question, hope it fixed

**Comment by bwiernik** [comment]
I don't really have the time to diagnose the issues here. If someone would like to take over maintenance of this plugin, please contact me.

At this point, the Zotero PDF reader is much faster than QuickLook ever was on any system. I woudl recommend just using the Zotero PDF reader. This would primarily require changing your keyboard shortcut to Enter/Return instead of spacebar.

**Comment by MasterHow** [comment]
looking forward to solve this, the beta version cannot downgrade to the lower version...

**Comment by bwiernik** [comment]
Please see my above reply.

**Comment by henusys** [comment]
I have an install method to resolve it. please look  https://www.bilibili.com/read/cv14492898

**Comment by bwiernik** [comment]
If that isn't spam, feel free to submit a PR. I am not actively working on this plugin

**Comment by skymagician** [comment]
> I have an install method to resolve it. please look https://www.bilibili.com/read/cv14492898

请问原来的插件是哪里出了问题？

**Comment by henusys** [comment]
> > I have an install method to resolve it. please look https://www.bilibili.com/read/cv14492898
> 
> 请问原来的插件是哪里出了问题？
在beta版zotero中空格键被占用了，因此需要修改quicklool的打开快捷键。另外该插件工作的关键是用到了bridge.exe，而该插件无法正确识别到bridge.exe的位置。

**Comment by henusys** [comment]
> If that isn't spam, feel free to submit a PR. I am not actively working on this plugin
It isn't spamming, just writing in Chinese. 
There are totally two problems. But it takes someone who knows JAVA to fix it.

1, The space key is occupied.
The Bata Zotero occupied the space key, and I just change the space key to ‘~’. In zoteroquicklook.js line 558, I changed as follows: 
 if (((event.keyCode==192 ) && !(event.ctrlKey || event.altKey || event.metaKey)) || (key == 'y' && event.metaKey && !(event.ctrlKey || event.altKey))) .

2, The QuickLook bridge executable (Win) can't be found correctly.

The problem may be at lines 36-48 in zoteroquicklook.js.  Unfortunately, I don't know JAVA so I can't solve it at the code level, and can only specify the location of 'bridge.exe' with the 'CustomViewCommand' parameter.

**Comment by vancleve** [comment]
> I don't really have the time to diagnose the issues here. If someone would like to take over maintenance of this plugin, please contact me.
> 
> At this point, the Zotero PDF reader is much faster than QuickLook ever was on any system. I woudl recommend just using the Zotero PDF reader. This would primarily require changing your keyboard shortcut to Enter/Return instead of spacebar.

At least on my MacBook Pro i9 w/ 32GB RAM, QuickLook is a bit faster; opening the PDF with the native reader takes a second or two. If that was indeed faster, then yes, I'd agree that using the native reader was preferable.

**Comment by lbc1027** [comment]
你好，邮件已收到，我会尽快处理，如有急事请联系13547906739

**Comment by skymagician** [comment]
> 你好，邮件已收到，我会尽快处理，如有急事请联系13547906739

太好了！终于有人来修复这个bug了！

**Comment by beaugunderson** [comment]
I have forked the project, my version is here (and only supports macOS): https://github.com/beaugunderson/ZoteroQuickLookNG

**Comment by bwiernik** [comment]
Please submit a pull request here or take over this project rather than forking the project
